netbout-web/src/test/java/com/netbout/rest/TkInboxTest.java

```diff
@@ -83,4 +91,43 @@ public void kicksAnUser() throws Exception {
             Matchers.containsString("you+kicked")
         );
     }
+
+    /**
+     * TkInbox can search messages.
+     * @throws Exception If there is some problem inside
+     */
+    @Test
+    public void searchesMessages() throws Exception {
+        final MkBase base = new MkBase();
+        final Bout bout = base.randomBout();
+        base.user(new URN(URN)).aliases().add(ALIAS);
+        bout.friends().invite(ALIAS);
+        final String hello = "hello";
+        final String world = "world";
+        bout.messages().post(hello);
+        bout.messages().post(world);
+        final String body = new RsPrint(
+            new TkAuth(
+                    new TkApp(base),
```

Comment:
@bdragan since we are creating a unit test for `TkInbox`, I'd argue we should use that instead of a full `TkApp`. What do you think?